src/pybind/mgr/rbd_support/module.py

```diff
@@ -673,9 +673,16 @@ def init_task_queue(self):
                 with self.module.rados.open_ioctx2(int(pool_id)) as ioctx:
                     self.load_task_queue(ioctx, pool_name)
 
-                    for namespace in rbd.RBD().namespace_list(ioctx):
-                        ioctx.set_namespace(namespace)
-                        self.load_task_queue(ioctx, pool_name)
+                    try:
+                        namespaces = rbd.RBD().namespace_list(ioctx)
+                    except rbd.OperationNotSupported:
+                        self.log.debug("Namespaces not supported")
+                        continue
+
+                    for namespace in namespaces:
+                            ioctx.set_namespace(namespace)
+                            self.load_task_queue(ioctx, pool_name)
```

Comment:
nit: why 8 spaces indentation?

Reply:
Good catch, I had put a `try` around the whole block but then didn't like it.